packages/fast-usdc/src/fast-usdc.contract.js

```diff
@@ -303,14 +306,15 @@ export const contract = async (zcf, privateArgs, zone, tools) => {
     }),
   );
 
-  const advancer = zone.makeOnce('Advancer', () =>
-    makeAdvancer({
-      borrower: poolKit.borrower,
-      notifier: settlerKit.notifier,
-      poolAccount,
-      settlementAddress,
-    }),
-  );
+  // we create a new Advancer on every upgrade. It does not contain precious state, but on each
+  // upgrade we must remember to call `advancer.setIntermediateRecipient()`
```

Comment:
Should we delete any `Advancer` that may exist in the zone? The main problem is that `makeOnce` doesn't have an "undo".

Reply:
That sounds like a good idea - how would we do that? 

Also - should there be any concern that this block causes the old Advancer to still get updates?
https://github.com/Agoric/agoric-sdk/blob/48d14ff79d3aca03113770c13c24f18f701f28c7/packages/fast-usdc/src/fast-usdc.contract.js#L315-L323

Reply:
> how would we do that?

I'm not sure the best way to do it, so let's not block on it, but we should keep track of it for a future cleanup.

> causes the old Advancer to still get updates?

My understanding is that this wires updates to the new advancer.

Reply:
Ok cool. I added a comment:

> // XXX delete `Advancer` that still may remain in zone after `update-settler-reference.core.js`